scripts/plot.py

```diff
@@ -145,3 +144,148 @@ def plot(directory_name):
         plt.grid()
         plt.savefig(directory / "GPU_Temp.png")
         plt.clf()
+
+
+def plot_comb(directory):
+    outputs, backends = load_all(directory)
+
+    plt.xlabel("Time(minutes)")
+    plt.ylabel("CPU Utilization (%)")
+    plt.grid()
+    for i, output in enumerate(outputs):
+        timestamp = output["_runtime"] / 60
+        cpu_utilization = outputs[i].get("system.cpu", [0] * len(timestamp))
+        plt.plot(timestamp, cpu_utilization, label=backends[i])
+    plt.legend(loc="upper left")
+    plt.savefig(directory / "CPU_Utilization.png")
+    plt.clf()
```

Comment:
Let's not copy and past a bunch of code like this over and over, but instead use the `matplotlib` subplots API and a function that ingests the axes. c.f. here for an example https://github.com/pyhf/tutorial-PyHEP-2020/blob/master/book/inference.ipynb

Reply:
I know we can avoid copy the same code for several times by using subplots. But in previous work, you told me that you expect plots stored into separate files rather than making use of subplot. Which choice will we choose?